src/manifestoo/addon_sorter.py

```
@@ -0,0 +1,50 @@
+from typing import Dict, Iterable, Set
+
+import graphlib
+import typer
+from graphlib import TopologicalSorter
+
+from manifestoo_core.addons_set import AddonsSet
+
+from . import echo
+
+
+class AddonSorter:
+    @staticmethod
+    def from_name(name: str) -> "AddonSorter":
+        if name == "alphabetical":
+            return AddonSorterAlphabetical()
+        elif name == "topological":
+            return AddonSorterTopological()
```

Fail early here if graphlib not available.

---

Lease test on sys.version_info it is more explicit